README.md

```diff
@@ -61,6 +61,10 @@ https://github.com/GENIVI/dlt-viewer
 
 [genivi-diagnostic-log-and-trace](https://lists.genivi.org/mailman/listinfo/genivi-diagnostic-log-and-trace_lists.genivi.org)
 
+## Additional Plugins
+
+https://github.com/svlad-90/DLT-Message-Analyzer
```

Comment:
Why not add it as submodule ?
If someone don't `git submodule update --init` he will not have it.
If yes, he will build it

Btw, as side effect, you will check plugin in CI as well.
I already tried it https://github.com/hannesa2/dlt-viewer/pull/3 but I failed

Reply:
@alexmucde